LOGBOOK.md

```diff
@@ -2,13 +2,17 @@
 
 Date based updates from the map based simulation team
 
+## 8 July 2024 - Released LAT map-based noise simulations
+
+Based on the LAT time-domain ``mss0002`` mission scale simulations, using the directional wavelet noise model of [`mnms`](https://github.com/simonsobs/mnms).
```

Comment:
I saw you have some releases https://github.com/simonsobs/mnms/tags of mnms, which version did you use for this?